languages/r36/template/bin/copy-cran-packages.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python
```

Comment:
we don't want a bin directory anymore
Also, I don't think we need the copy script anymore (more in a follow up comment)

Reply:
yea Ill throw all of this into the context